AutomatedTesting/Gem/PythonTests/Terrain/EditorScripts/Terrain_SupportsPhysics.py

```diff
@@ -0,0 +1,164 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+#fmt: off
+class Tests():
+    create_terrain_spawner_entity               = ("Terrain_spawner_entity created successfully", "Failed to create terrain_spawner_entity")
+    create_height_provider_entity               = ("Height_provider_entity created successfully", "Failed to create height_provider_entity")
+    create_test_ball                            = ("Ball created successfully",    "Failed to create Ball")
+    box_dimensions_changed                      = ("Aabb dimensions changed successfully", "Failed change Aabb dimensions")
+    shape_changed                               = ("Shape changed successfully", "Failed Shape change")
+    entity_added                                = ("Entity added successfully", "Failed Entity add")
+    frequency_changed                           = ("Frequency changed successfully", "Failed Frequency change")
+    shape_set                                   = ("Shape set to Sphere successfully", "Failed to set Sphere shape")
+    test_collision                              = ("Ball collided with terrain", "Ball failed to collide with terrain")
+    no_errors_and_warnings_found                = ("No errors and warnings found", "Found errors and warnings")
+#fmt: on
+
+def Terrain_SupportsPhysics():
+    """
+    Summary:
+    Test aspects of the TerrainHeightGradientList through the BehaviorContext and the Property Tree.
+
+    Test Steps:
+    Expected Behavior:
+    The Editor is stable there are no warnings or errors.
+
+    Test Steps:
+     1) Load the base level
+     2) Create 2 test entities, one parent at 512.0, 512.0, 50.0 and one child at the default position and add the required components
+     2a) Create a ball at 600.0, 600.0, 46.0 - This position is bot too high over the heighfield so will collide in a reasonable time
+     3) Start the Tracer to catch any errors and warnings
+     4) Change the Axis Aligned Box Shape dimensions
+     5) Set the Vegetation Shape reference to TestEntity1
+     6) Set the FastNoise gradient frequency to 0.01
+     7) Set the Gradient List to TestEntity2
+     8) Set the PhysX Collider to Sphere mode
+     9) Disable and Enable the Terrain Gradient List so that it is recognised
```

Comment:
Is this something a user has to do in the editor or is it only required in the test environment for some reason? If you do have to do it in the editor, doesn't seem like an ideal workflow.

Reply:
Will fix.